src/current/_includes/v24.1/misc/force-index-selection.md

```diff
@@ -62,6 +62,21 @@ you can check the scan direction with:
 (2 rows)
 ~~~
 
+#### Force inverted index scan
+
+To force a scan of any [inverted index]({% link {{ page.version.version }}/inverted-indexes.md %}) of the hinted table:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM table@{FORCE_INVERTED_INDEX};
+~~~
+
+The `FORCE_INVERTED_INDEX` hint does not allow specifying an inverted index. If no query plan can be generated, the query will result in the error:
```

Comment:
I don't understand this sentence, but maybe it's just because I'm not familiar with the optimizer or indexes. I would expect the session variable to do exactly what it says it doesn't do.

Reply:
Yeah, it's because unlike the other hints, this one can't be appended with a specific index name to scan. I think that should be clear on this page, since the other examples show this.